docs/extensions/services/accessing-files.md

```diff
@@ -43,6 +43,41 @@ router.post('/', async (req, res) => {
 });
 ```
 
+## Upload a File
+
+```js
+router.post('/', async (req, res, next) => {
+  const filesService = new FilesService({
+    schema: await getSchema(),
+    accountability: req.accountability,
+  });
+
+  const busboy = Busboy({ headers: req.headers });
```

Comment:
Considering `Busboy` is not a globally available function it may be good to mention it needs to be imported `import Busboy from 'busboy';` and that it's not needed to compile `busboy` into your extension because it works as "external dependency".